content/sensu-core/0.29/guides/securing-sensu.md

```diff
@@ -0,0 +1,303 @@
+---
+title: "Securing Sensu"
+description: "Strategies and best practices for securing Sensu"
+version: "0.29"
+weight: 8
+next: ../securing-rabbitmq-guide
+menu:
+  sensu-core-0.29:
+    parent: guides
+---
+
+Securing Sensu is a multifaceted process that requires several different components to be secured in order to properly harden your Sensu deployment. In this guide, we'll walk you through the best practices and strategies for securing Sensu. By the end of the guide, you should have a thorough understanding of what goes into securing all of the pieces that make up a Sensu deployment, including:
+
+* How to secure your Sensu clients
+* How to secure your Sensu Server/API/Enterprise instance(s)
+* How to secure Uchiwa/the Sensu Enterprise Dashboard
+
+We'll also walk through securing the additional components like RabbitMQ and Redis in the guides following this one.
+
+# Objectives
+
+We'll cover the following in this guide:
+
+* [Securing Sensu](#securing-sensu-clients)
+  * [The `client_signature` Attribute](#the-clientsignature-attribute)
+  * [The `redact` Attribute](#the-redact-attribute)
+  * [SSL/TLS Encryption for Transport Communication](#client-ssl-tls)
+* [Securing Sensu Server/API](#securing-sensu-server)

```

Comment:
IMO the SSL/TLS recommendations for transport (AMQPS for sensu-client, sensu-server, sensu-api and/or sensu-enterprise) and web UI (HTTPS for sensu-api+uchiwa or sensu-enterprise+sensu-enterprise-dashboard) should all go under the same "Encrypting communications using SSL/TLS"

Reply:
✔️